README.md

```diff
@@ -7,4 +7,7 @@ Uses [WISVCH/bootstrap-theme](https://github.com/WISVCH/bootstrap-theme) for sty
 
 1. Install [IntelliJ IDEA Professional](https://www.jetbrains.com/idea/) ([free for students](https://www.jetbrains.com/student/))
 2. [Import project from Gradle model](https://www.jetbrains.com/idea/help/importing-project-from-gradle-model.html)
-3. Start the project by right-clicking `EventsApplication` and clicking 'Run'
+3. [Install the lombok plugin](https://github.com/mplushnikov/lombok-intellij-plugin)
+4. Duplicate the file config/application.example.properties and change its name to config/application.properties
```

Comment:
I would restructure this sentence to a copy action instead of duplicate. Makes it a bit more clear and it sounds less complicated.